docs/design/AWS_API_Compatibility.md

```
@@ -71,16 +74,32 @@ _* Note that it is also possible for actions to be supported but absent because
+| **STS API** | Session Tokens | AssumeRole           | ✅ | ❌ | |
+| **IAM API** | Users          | CreateUser           | ✅ | ✅ | |
+|             | Users          | GetUser              | ✅ | ✅ | |
+|             | Users          | UpdateUser           | ✅ | ✅ | |
+|             | Users          | DeleteUser           | ✅ | ✅ | |
+|             | Users          | ListUsers            | ✅ | ✅ | *No pagination support|
+|             | Access Keys    | CreateAccessKey      | ✅ | ✅ | |
+|             | Access Keys    | GetAccessKeyLastUsed | ✅ | ✅ | *Partially implemented|
+|             | Access Keys    | UpdateAccessKey      | ✅ | ✅ | |
+|             | Access Keys    | DeleteAccessKey      | ✅ | ✅ | |
+|             | Access Keys    | ListAccessKeys       | ✅ | ✅ | *No pagination support|
+|             | Tag User       | TagUser              | ✅ | ❌ | | |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 6; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)